packages/generators/remove-generator.js

```diff
@@ -1,3 +1,119 @@
 const Generator = require("yeoman-generator");
+const path = require("path");
+const fs = require("fs");
+const { List } = require("@webpack-cli/webpack-scaffold");
 
-module.exports = class RemoveGenerator extends Generator {};
+const PROP_TYPES = require("@webpack-cli/utils/prop-types");
+
+/**
+ *
+ * Generator for removing properties
+ * @class	RemoveGenerator
+ * @extends	Generator
+ * @returns	{Void} After execution, transforms are triggered
+ *
+ */
+
+module.exports = class RemoveGenerator extends Generator {
+	constructor(args, opts) {
+		super(args, opts);
+		this.configuration = {
+			config: {
+				webpackOptions: {},
+			}
+		};
+
+		let configPath = path.resolve(process.cwd(), "webpack.config.js");
+		const webpackConfigExists = fs.existsSync(configPath);
+		if (!webpackConfigExists) {
+			configPath = null;
+			// end the generator stating webpack config not found or to specify the config
+		}
+		this.webpackOptions = require(configPath);
```

Comment:
This is neat, but it will throw on packages that uses plugins. Maybe a JSON serialize would be better. If you did the same approach with `add`, the strings from the config needs to be padded in order for the ast to construct correctly. 

It's unclear what the comment is pointing to, what is it suppose to comment?